.internal-ci/helm/fog-ingest/templates/fog-ingest-statefulset.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   labels:
     {{- include "fogIngest.labels" . | nindent 4 }}
 spec:
+  podManagementPolicy: Parallel
```

Comment:
_NIIIICE_

Reply:
Small nit.  This was defined in values.yaml for fog-ledger. Do we need/want to do for ingest?  Won't really matter in the long run unless we think we might need to override this in the future.